File: client/src/app/data/package/package-details.ts
import { Customer } from "../customer/customer"

export class PackageDetails {
    name: String
    price: String
    endDate: string
    purchaseDate: string
    usedEntries: number
    remainingEntries: number
    duration: number
    owners: Customer[]
    paid: boolean

    end_date: Date
    purchase_date: Date
}

File: server/src/main/java/pl/kodokan/fcp/server/entrance/controller/PackageDetails.java
package pl.kodokan.fcp.server.entrance.controller;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@Builder
public class PackageDetails {
    private String name;
    private BigDecimal price;
    private boolean isPaid;
    private LocalDateTime endDate;
    private LocalDateTime purchaseDate;
    private Integer usedEntries;
    private Integer remainingEntries;
    private Long duration;
    private List<SimpleCustomerDetails> owners;

    private class SimpleCustomerDetails {
        private String customerId;
        private String firstName;
        private String lastName;
    }

}
